### ISSUE TITLE
API call not returning image, just a tiny blank image

### ISSUE THREAD

**Issue by davecoffin** [nicholasgubbins/Serverless-Image-Resizer#21]
Hey there, I am super pumped to use this as it is exactly what I need. I have installed serveless and deployed my service without any issues, but I'm confused as to exactly what I should be expecting. According to your documentation, if the image requested doesn't exist in the S3 bucket it will create it AND return it. If it does it will just return it. But I am experiencing something odd... I have an image in my bucket called noah2invitation.jpg, and if I request that image with my serverless-created api url, I get a small blank image: 
Heres a screenshot: https://cl.ly/p5bO
https://e5eqvjclo9.execute-api.us-east-1.amazonaws.com/dev/noah2invitation.jpg

Here is the image in my bucket: 
https://s3.amazonaws.com/dev-dailynanny-photos/noah2invitation.jpg

Then, if I request a transformation: 
https://e5eqvjclo9.execute-api.us-east-1.amazonaws.com/dev/noah2invitation.jpg?w=100&h=100

I get the same small blank image, but it does create the image in my bucket: 
https://s3.amazonaws.com/dev-dailynanny-photos/noah2invitation.jpg%3Fh%3D100%26w%3D100

Why arent the API calls returning the images?

Thanks, and thanks for making this!

**Comment by marcusmolchany** [comment]
Hey @davecoffin, thanks for the detail here. Have you set up the [binary support with api gateway](https://github.com/nicholasgubbins/Serverless-Image-Resizer#api-gateway-binary-support)?

**Comment by davecoffin** [comment]
Yes I have, but the flow has changed since the documentation was last authored (in fact, I havent read any documentation that references the current state of their UI, it must have been updated recently?). Here's what I have done, please let me know if this is enough:

https://cl.ly/p62x

It appears to be automatically deployed, I didn't find anything anywhere to "deploy" the api.

**Comment by davecoffin** [comment]
@marcusmolchany Any thoughts on this?

**Comment by pabloheimplatz** [comment]
@davecoffin just click on the top left on the name of your API gateway function (dev-Image-Resizer). There you will find the "Action" button where you can deploy the API. 
After that change it works for me.
